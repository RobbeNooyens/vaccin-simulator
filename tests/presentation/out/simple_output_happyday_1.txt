Hub (178000): 
  -> Center 1 (0 vaccins)
  -> Center 2 (1189 vaccins)
  -> Center 3 (0 vaccins)

Center 1: 0 gevaccineerd, nog 5043 inwoners niet gevaccineerd
Center 2: 8011 gevaccineerd, nog 0 inwoners niet gevaccineerd
Center 3: 0 gevaccineerd, nog 3021 inwoners niet gevaccineerd
