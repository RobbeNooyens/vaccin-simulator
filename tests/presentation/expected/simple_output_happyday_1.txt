Hub (409000): 
  -> Center 1 (3560 vaccins)
  -> Center 2 (1200 vaccins)
  -> Center 3 (4000 vaccins)
Hub (248000): 
  -> Center 3 (4000 vaccins)
  -> Center 4 (6180 vaccins)
  -> Center 5 (7424 vaccins)

Center 1: 154020 gevaccineerd, nog 0 inwoners niet gevaccineerd
Center 2: 235900 gevaccineerd, nog 549320 inwoners niet gevaccineerd
Center 3: 504660 gevaccineerd, nog 441340 inwoners niet gevaccineerd
Center 4: 24300 gevaccineerd, nog 0 inwoners niet gevaccineerd
Center 5: 123456 gevaccineerd, nog 0 inwoners niet gevaccineerd
